docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: model
+title: Sentiment Analysis of Vietnamese texts
+author: John Snow Labs
+name: classifierdl_distilbert_sentiment
+date: 2022-02-09
+tags: [vietnamese, sentiment, vi, open_source]
+task: Text Classification
+language: vi
+edition: Spark NLP 3.4.0
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model identifies Positive or Negative sentiments in Vietnamese texts.
+
+## Predicted Entities
+
+`POSITIVE`, `NEGATIVE`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/classifierdl_distilbert_sentiment_vi_3.4.0_3.0_1644408533716.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document = DocumentAssembler()\
+    .setInputCol("text")\
+    .setOutputCol("document")\
+    .setCleanupMode("shrink")
+
+tokenizer = Tokenizer() \
+      .setInputCols(["document"]) \
+      .setOutputCol("token")
+    
+normalizer = Normalizer() \
+      .setInputCols(["token"]) \
+      .setOutputCol("normalized")
+
+lemmatizer = LemmatizerModel.pretrained("lemma", "vi") \
+        .setInputCols(["normalized"]) \
+        .setOutputCol("lemma")
+
+distilbert = DistilBertEmbeddings.pretrained("distilbert_base_cased", "vi")\
+  .setInputCols(["document",'token'])\
+  .setOutputCol("embeddings")\
+  .setCaseSensitive(False)
+
+embeddingsSentence = SentenceEmbeddings() \
+      .setInputCols(["document", "embeddings"]) \
+      .setOutputCol("sentence_embeddings") \
+      .setPoolingStrategy("AVERAGE")
+
+sentimentClassifier = ClassifierDLModel.pretrained('classifierdl_distilbert_sentiment', 'vi') \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+vi_sentiment_pipeline = Pipeline(stages=[document, tokenizer, normalizer, lemmatizer, distilbert, embeddingsSentence, sentimentClassifier])
+
+light_pipeline = LightPipeline(vi_sentiment_pipeline.fit(spark.createDataFrame([['']]).toDF("text")))
+result = light_pipeline.annotate("Chất cotton siêu đẹp mịn mát.")
```

Comment:
Is this sentence a real one from the dataset or any other native internet resource?

Reply:
@josejuanmartinez 

it is taken from train data set


![image](https://user-images.githubusercontent.com/69839085/153239516-fc341cc2-8167-41c6-88df-6cadd26136dd.png)